api/nodes/views.py

```diff
@@ -3357,6 +3357,13 @@ class NodePreprintsList(JSONAPIBaseView, generics.ListAPIView, NodeMixin, NodePr
     view_category = 'nodes'
     view_name = 'node-preprints'
 
+    def postprocess_query_param(self, key, field_name, operation):
+        if field_name == 'provider':
+            operation['source_field_name'] = 'provider___id'
+
+        if field_name == 'id':
+            operation['source_field_name'] = '_guids___id'
```

Comment:
I think Chris is right, though I'm not sure why the tests would be passing if this field is incorrect. Are the tests testing what we think they are testing? @alexschiller Please investigate.